### ISSUE TITLE
Error when running the basic usage for Node.js

### ISSUE THREAD

**Issue by lucj** [balderdashy/sails.io.js#32]
When following the example for Node.js 'Basic Usage', I got the following error message

/Users/luc/websocket2/node_modules/sails.io.js/sails.io.js:377
    Socket.prototype.get = function(url, data, cb) {
          ^
TypeError: Cannot read property 'prototype' of undefined
    at SailsIOClient (/Users/luc/websocket2/node_modules/sails.io.js/sails.io.js:377:11)
    at Object.<anonymous> (/Users/luc/websocket2/index.js:6:10)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3

**Comment by ws6** [comment]
any updates? sails.io.js client work under nodejs would be great.

**Comment by lucj** [comment]
Did you try with socket.io-client v 0.9.17 in basic usage with node.js part ?

**Comment by ws6** [comment]
no luck

**Comment by sgress454** [comment]
Make sure when using `sails.io.js` that you're using the version intended for your version of Sails.  The easiest way is to grab it from the `assets/js/dependencies` folder of your Sails app, or if you've removed it, do `sails new someApp` to generate a new Sails app to grab the file from.

**Comment by bmwertman** [comment]
I'm having the same issue @sgress454 that you answer in this SO post http://stackoverflow.com/questions/28138152/connect-from-a-client-to-a-sails-based-server?lq=1 I'm on sails 0.11.0 and used sails new to make a new app and get the sails.io.js file. But the issue persists. Any ideas?

**Comment by sailsbot** [comment]
Thanks for posting, @lucj.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
